crates/rolldown/src/ast_scanner/side_effect_detector/utils.rs

```diff
@@ -227,6 +227,12 @@ pub fn is_primitive_literal(scope: &AstScopes, expr: &Expression) -> bool {
     | Expression::NumericLiteral(_)
     | Expression::StringLiteral(_)
     | Expression::BigIntLiteral(_) => true,
+    // oxc number literal did not included `+1` and `-1`
```

Comment:
Include `+1` / `-1`.
```suggestion
    // oxc number literal did not included `+1` and `-1`
```